modules/govuk/manifests/apps/rummager/rabbitmq.pp

```diff
@@ -34,23 +38,32 @@
     amqp_queue    => 'rummager_to_be_indexed',
     routing_key   => '*.links',
     durable       => true,
-  } ->
```

Comment:
Just to check - did you remove the `->` because the resources don't actually need to be created in a specific order?

Reply:
I don't think they do, no. I actually introduced those arrows a couple of days ago when refactoring. The user permissions reference the queues, but they're just regular expressions so the queues shouldn't need to exist to create the users.